examples/hf_bert.py

```diff
@@ -0,0 +1,64 @@
+import prewl, json
+
+#-----------------------------------------#
+
+# Optionally use a config.json
+#prewl.configure("config.json")
+
+# Dynamically detect if the argument is json or a file with json
+prewl.configure({
+    'backend': {
+        'service': 'bert-base-uncased',
+        'remote': True,
+        'token': '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' # Use your huggingface token
+    },
+    'newline-delimited': True, # Should a new line indicate the end of the completion
+    'classes': None, # defaults to None
+    'max-length': 3, # output response upper length limit
+    'gpu': True, # True or the GPU number you would like to use (if local). False (or -1 if local) indicates CPU,
+    'resp_parser': 'mask_fill_parser'
+})
+
+#-----------------------------------------#
+
+PROMPTS = """
```

Comment:
Here you can see the prompts for this model have only keys, no values. This is unlike the sentiment analysis example.